lib/matplotlib/backends/backend_gtk.py

```diff
@@ -1003,13 +999,13 @@ def on_combobox_lineprops_changed(self, item):
         if marker is None: marker = 'None'
         self.cbox_markers.set_active(self.markerd[marker])
 
-        r,g,b = colorConverter.to_rgb(line.get_color())
-        color = gtk.gdk.Color(*[int(val*65535) for val in (r,g,b)])
+        rgba = mcolors.to_rgba(line.get_color())
```

Comment:
Does GTK have the concept of an RGBA color?  (Not according to http://www.pygtk.org/pygtk2reference/class-gdkcolor.html, but I don't known much about GTK...)
Frankly I'd rather not expose `to_rgb` as a public API anymore, it's too easy to lose information on the way...